Makefile

```diff
@@ -96,25 +96,28 @@ help: ## Show this help screen
 .PHONY: all
 all: manager ## Build all binaries
 
-$(GO-LICENSES):
+$(TIMESTAMPS_DIR)/go-licenses-installed:
 	go install github.com/google/go-licenses@latest
+	@mkdir -p $(TIMESTAMPS_DIR) && touch $@
+
+go-licenses: $(TIMESTAMPS_DIR)/go-licenses-installed
 
-licenses.csv: $(GO_SOURCES) go.mod ## Track licenses in a CSV file
+licenses.csv: $(TIMESTAMPS_DIR)/go-licenses-installed go.mod ## Track licenses in a CSV file
 	@echo "Tracking licenses into file $@"
 	@echo "========================================"
-	$(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > $@
+	GOOS=linux GOARCH=amd64 $(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > $@
 
 .PHONY: check-licenses
 check-licenses: licenses.csv  ## Check licenses
 	@echo "Checking licenses not to be: $(DISALLOWED_LICENSES)"
 	@echo "============================================"
-	$(GO_LICENSES) check --include_tests $(BASE_GO_PACKAGE)/... \
+	GOOS=linux GOARCH=amd64 $(GO_LICENSES) check --include_tests $(BASE_GO_PACKAGE)/... \
 	--disallowed_types $(DISALLOWED_LICENSES)
```

Comment:
The `DISALLOWED_LICENSES` seems not defined. Is that intentional?

Reply:
will double check, thanks for the heads up

Reply:
It is defined in line 84

Reply:
Perfect! Thanks!